build/deploy/db_schemas/rid.libsonnet

```diff
@@ -1,19 +1,16 @@
 {
   data:{
-    "000001_create_isa_subs_tables.down.sql": importstr "rid/000001_create_isa_subs_tables.down.sql",
-    "000001_create_isa_subs_tables.up.sql": importstr "rid/000001_create_isa_subs_tables.up.sql",
-    "000002_inverted_index.down.sql": importstr "rid/000002_inverted_index.down.sql",
-    "000002_inverted_index.up.sql": importstr "rid/000002_inverted_index.up.sql",
-    "000003_migrate_data_to_index.down.sql": importstr "rid/000003_migrate_data_to_index.down.sql",
-    "000003_migrate_data_to_index.up.sql": importstr "rid/000003_migrate_data_to_index.up.sql",
-    "000004_drop_cells_table.down.sql": importstr "rid/000004_drop_cells_table.down.sql",
-    "000004_drop_cells_table.up.sql": importstr "rid/000004_drop_cells_table.up.sql",
-    "000005_bump_version.down.sql": importstr "rid/000005_bump_version.down.sql",
-    "000005_bump_version.up.sql": importstr "rid/000005_bump_version.up.sql",
-    "000006_add_writer_column.down.sql": importstr "rid/000006_add_writer_column.down.sql",
-    "000006_add_writer_column.up.sql": importstr "rid/000006_add_writer_column.up.sql",
-    "000007_add_index_by_time_subscriptions.down.sql": importstr "rid/000007_add_index_by_time_subscriptions.down.sql",
-    "000007_add_index_by_time_subscriptions.up.sql": importstr "rid/000007_add_index_by_time_subscriptions.up.sql",
-    "000008_db_rename_to_rid.up.sql": importstr "rid/000008_db_rename_to_rid.up.sql"
+    "upto-v1.0.0-remove_isa_subs_tables.sql": importstr "rid/upto-v1.0.0-remove_isa_subs_tables.sql",
+    "upto-v2.0.0-add_inverted_indices.sql": importstr "rid/upto-v2.0.0-add_inverted_indices.sql",
+    "upto-v3.0.0-no_change.sql": importstr "rid/upto-v3.0.0-no_change.sql",
+    "upto-v3.1.0-add_writer_column.sql": importstr "rid/upto-v3.1.0-add_writer_column.sql",
+    "upto-v3.1.1-add_index_by_time_subscriptions.sql": importstr "rid/upto-v3.1.1-add_index_by_time_subscriptions.sql",
+    "upto-v4.0.0-rename_defaultdb_to_rid.sql": importstr "rid/upto-v4.0.0-rename_defaultdb_to_rid.sql",
+    "downfrom-v4.0.0-move_rid_to_defaultdb.sql": importstr "rid/downfrom-v4.0.0-move_rid_to_defaultdb.sql",
+    "downfrom-v3.1.1-remove_index_by_time_subscriptions.sql": importstr "rid/downfrom-v3.1.1-remove_index_by_time_subscriptions.sql",
+    "downfrom-v3.1.0-remove_writer_column.sql": importstr "rid/downfrom-v3.1.0-remove_writer_column.sql",
+    "downfrom-v3.0.0-no_change.sql": importstr "rid/downfrom-v3.0.0-no_change.sql",
+    "downfrom-v2.0.0-remove_inverted_indices.sql": importstr "rid/downfrom-v2.0.0-remove_inverted_indices.sql",
+    "downfrom-v1.0.0-create_isa_subs_tables.sql": importstr "rid/downfrom-v1.0.0-create_isa_subs_tables.sql",
```

Comment:
The name of the file seems wrong. Shouldn't it be downfrom-v1.0.0-remove_isa_subs_tables.sql ?

Reply:
Good catch!  Fixed.